app/controllers/application_controller.rb

```diff
@@ -5,11 +5,17 @@ def index
     @locations_full = Location.all.sort_by{|location| hav_distance([location.latitude.to_f, location.longitude.to_f], @current_location)}
   end
 
+  def tips
```

Comment:
Is it typical in Rails to have functions with no content? (Maybe! I don't recall; just checking)